src/types.ts

```diff
@@ -353,3 +352,7 @@ export type Error = {
   stack?: string,
   value?: any
 };
+
+export type UIOptions = {
+  slowMo?: number;
```

Comment:
showMouseCursor and showKeyboardPresses will go in here in a follow up patch.

Reply:
is `showMouseCursor` going to behave like this: https://github.com/puppeteer/puppeteer/issues/4378#issuecomment-499726973 ?
Is it possible to interpolate the mouse cursor between `.click`s?